### PR TITLE
feat: add payloads to update native aDI bridge adapters

### PR DESCRIPTION
**Pull request by sendra**
Updates the native bridge adapters of aDI for the paths:
- Eth -> Pol
- Eth -> Arb
- Eth -> Op
- Eth -> Metis
- Eth -> Gnosis
- Eth -> Eth
- Eth -> Scroll
- Pol -> Eth
- Eth -> Base

### CODE REVIEW

File: src/20240322_Multi_NativeBridgeAdaptersUpdate/AaveV3Ethereum_NativeBridgeAdaptersUpdate_20240322.t.sol [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import './BaseTest.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {AaveV3Ethereum_NativeBridgeAdaptersUpdate_20240322} from './AaveV3Ethereum_NativeBridgeAdaptersUpdate_20240322.sol';
+
+/**
+ * @dev Test for AaveV3Ethereum_NativeBridgeAdaptersUpdate_20240322
+ * command: make test-contract filter=AaveV3Ethereum_NativeBridgeAdaptersUpdate_20240322
+ */
+contract AaveV3Ethereum_NativeBridgeAdaptersUpdate_20240322_Test is BaseTest {
```
Comment (kyzia551) [inline src/20240322_Multi_NativeBridgeAdaptersUpdate/AaveV3Ethereum_NativeBridgeAdaptersUpdate_20240322.t.sol:12]: Should it be more extensive?

**Review by kyzia551 — DISMISSED** [review]

File: src/20240322_Multi_NativeBridgeAdaptersUpdate/NativeBridgeAdaptersUpdate.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Native bridge adapters update"
+author: "BGD Labs @bgdlabs"
+discussions: ""
+---
+
+## Simple Summary
+
+This proposal updates the Native bridge adapters used on a.DI to connect Ethereum with Polygon, Arbitrum, Optimism, Base, Metis, Gnosis, Scroll, Base and the Ethereum SameChain adapter.
```
Comment (eboadom) [inline src/20240322_Multi_NativeBridgeAdaptersUpdate/NativeBridgeAdaptersUpdate.md:9]:
```suggestion
This proposal updates the Native bridge adapters used on a.DI, to add extra naming metadata and make the gas limit configurations more granular.
```

File: src/20240322_Multi_NativeBridgeAdaptersUpdate/NativeBridgeAdaptersUpdate.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Motivation
+
+The main motivation of this proposal is to bring the Native bridge adapters up to date
```
Comment (eboadom) [inline src/20240322_Multi_NativeBridgeAdaptersUpdate/NativeBridgeAdaptersUpdate.md:13]:
```suggestion
For the past month with a.DI in production, we have noticed small improvements due to the specific mechanics of each bridge provider. Additionally, as we improve the off-chain systems (e.g. monitoring) surrounding a.DI, sometimes it is simply better to add features to the smart contracts to facilitate this off-chain infrastructure.
```

**Review by kyzia551 — DISMISSED** [review]

File: src/20240322_Multi_NativeBridgeAdaptersUpdate/NativeBridgeAdaptersUpdate.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Native bridge adapters update"
+author: "BGD Labs @bgdlabs"
+discussions: ""
```
Comment (sendra) [inline src/20240322_Multi_NativeBridgeAdaptersUpdate/NativeBridgeAdaptersUpdate.md:4]:
```suggestion
discussions: "https://governance.aave.com/t/bgd-technical-maintenance-proposals/15274/26"
```

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**